### PR TITLE
fix(Scripts/item); Glyph of Kilrogg

### PR DESCRIPTION
**Pull request by grimgravy**
## Changes Proposed:
-  Fix: speed run
-  Fix: flight speed

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/12300

## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Gultask** [comment]
Works. Movement speed while flying is significantly faster with the fix.

**Comment by acidmanifesto** [comment]
Merge Conflicts

File: src/server/game/Spells/SpellInfoCorrections.cpp [hunk 1]
```diff
@@ -4421,6 +4421,16 @@ void SpellMgr::LoadSpellInfoCorrections()
...
 
+    // Eye of Kilrogg Passive (DND)
+    ApplySpellFix({ 58083 }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->Effects[EFFECT_2].Effect = SPELL_EFFECT_APPLY_AURA;
+        spellInfo->Effects[EFFECT_2].BasePoints = 49;
+        spellInfo->Effects[EFFECT_2].DieSides = 1;
+        spellInfo->Effects[EFFECT_2].TargetA = SpellImplicitTargetInfo(TARGET_UNIT_CASTER);
+        spellInfo->Effects[EFFECT_2].ApplyAuraName = SPELL_AURA_MOD_INCREASE_SPEED;
+    });
```
Comment (Kitzunu) [inline src/server/game/Spells/SpellInfoCorrections.cpp:4432]: Why

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: src/server/game/Spells/SpellInfoCorrections.cpp (hunk 1 above)
Comment (grimgravy) [reply src/server/game/Spells/SpellInfoCorrections.cpp:4432]: @Kitzunu, the Spell Glyph of Kilrogg  does not work, as it also serves as a basis in the script

**Comment by UltraNix** [comment]
Superseded by #13802

**Pull request closed without merging**